client-v2/package.json

```diff
@@ -5,8 +5,8 @@
   "scripts": {
     "start": "expo start",
     "reset-project": "node ./scripts/reset-project.js",
-    "android": "expo start --android",
-    "ios": "expo start --ios",
+    "android": "expo run:android",
```

Comment:
Will this be slow vs start?

Reply:
side effect of `expo prebuild`: https://docs.expo.dev/workflow/prebuild/#side-effects

reverting